app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt

```diff
@@ -63,20 +60,6 @@ class AddonsManagementFragment : Fragment(R.layout.fragment_add_ons_management),
         findPreviousDialogFragment()?.let { dialog ->
             dialog.onPositiveButtonClicked = onPositiveButtonClicked
         }
-
-        scope = requireContext().components.core.store.flowScoped { flow ->
-            flow.ifChanged { it.extensions }
-                .collect { _ ->
-                    view?.let {
-                        bindRecyclerView(it)
```

Comment:
I found we were refreshing the whole list multiple times. When we install an add-on, we are getting notified 
one times on `InstallWebExtensionAction` and two times on `UpdateBrowserAction`